HomeWork/Singleton/Loger.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Singleton
+{
+    public class Loger
+    {
+        private static readonly Lazy<Loger> _lazy = new Lazy<Loger>(() => new Loger());
+        public string Path { get; }
+
+        private Loger()
+        {
+            Console.Write("Введите путь к файлу: ");
```

Comment:
For real task, please, try to avoid such things. Models, services, etc. should not contain references to console, forms, windows, etc. 
In our lessons examples we put Console.WL just for demonstrating.